scripts/get-changed-urls.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# This script lists URLs changed by the target PR. You can jump to target page quickly by clicking those.
+# Syntax: get-changed-urls.sh <PR number> [<page top URL>]
+# Example: 
+# $ get-changed-urls.sh 28542
+# http://localhost:1313/ja/docs/concepts/configuration/manage-resources-containers
+# http://localhost:1313/ja/docs/concepts/policy/limit-range
+# http://localhost:1313/ja/docs/concepts/policy/resource-quotas
+# http://localhost:1313/ja/docs/reference/command-line-tools-reference/feature-gates
+# http://localhost:1313/ja/docs/setup/best-practices/cluster-large
+#
+# If no <page top URL> is passed, it will assume "http://localhost:1313".
+# It's the default endpoint for "make container-serve" or "make serve".
+# Please look at the README(https://github.com/kubernetes/website#using-this-repository) to serve endpoint.
+# Or you can pass any strings to <page top URL>.
+# You can get URLs for upstream by passing <page top URL> as "https://kubernetes.io".
+
+set -e
+
+if [[ ${1} == "" ]]
+then
+	echo "This script prints all URLs updated by target PR number."
+	echo "You can jump to updated pages quickly when PR review."
+	echo "This is intended for use with \"make container-serve\"."
```

Comment:
nit: s/use/using/

Reply:
thank you for comment.
I've fixed this.